tools/essence-feature-usage-stats/.env

```diff
@@ -0,0 +1,11 @@
+ESSENCE_DIR=/home/mayday/Coding/VIP/EssenceCatalog
```

Comment:
are these local paths still needed?

Reply:
No, the whole .env file is just for local testing. All the necessary env vars are in the GitHub Action

Reply:
Should we remove them from the repo then? As they are, they won't work on anybody else's computer.

Reply:
True! I have removed the file, just waiting for the CI to complete and the commit will appear here